app/models/status.rb

```diff
@@ -6,7 +6,7 @@ class Status < ApplicationRecord
   include Streamable
   include Cacheable
 
-  enum visibility: [:public, :unlisted, :private, :direct], _suffix: :visibility
+  enum visibility: [:public, :local, :unlisted, :private, :direct], _suffix: :visibility
```

Comment:
I believe adding this enum here will shift the integer of the other values right, for example:

public: 1, unlisted: 2, private: 3, direct: 4

will become

public: 1, local: 2, unlisted: 3, private: 4, direct: 5

This will cause the visibility of all currently existing statuses in the db to change!

Unfortunately, you'll have to put `local` at the end